### PR TITLE
Handle image loading error (in non-detached window)

### PR DESCRIPTION
**Pull request by JaySunSyn**
Please merge https://github.com/Neovici/img-pan-zoom/pull/1 first.

### CODE REVIEW

File: cosmoz-image-viewer.js [hunk 1]
```diff
@@ -291,19 +291,23 @@
 		get imageOverlay() {
 			return imageOverlay;
 		},
+
+		get carousel() {
+			return this.$$('#carousel');
```
Comment (nomego) [inline cosmoz-image-viewer.js]: Why not `this.$.carousel` ?

Comment (JaySunSyn) [reply cosmoz-image-viewer.js]: In the tests, I prefer to reference "public" properties where possible rather than querySelect the carousel.

**Comment by JaySunSyn** [comment]
Yes, if we would always allow to zoom, we can drop iron image.

**Comment by JaySunSyn** [comment]
I'll try to get rid of iron-image in another PR

**Comment by JaySunSyn** [comment]
@nomego Build finally passes, pls merge.

**Pull request merged**